README.md

```diff
@@ -75,3 +75,68 @@ This project is still in early stages but future (exciting!) plans include:
 [issue94]: https://github.com/jkcfg/jk/issues/94
 [issue110]: https://github.com/jkcfg/jk/issues/110
 [issueHermeticity]: https://github.com/jkcfg/jk/issues?q=is%3Aissue+is%3Aopen+label%3Atopic%2Fhermeticity
+
+## Contributing
+When contributing to this repository, please first discuss the change you wish to make via issue with the owners of this repository before making a change.
+
+### Setup
+The `jk` executable itself is written in [Go](https://golang.org), but the JavaScript part of this project requires [NodeJS](https://nodejs.org).
+
+**Prerequisites:**
+* `go` 1.11+ (modules support)
+* `nodejs`, `npm`
+* `make`
+
+First off, clone this repository:
+```bash
+$ git clone https://github.com/jkcfg/jk.git
+# or with hub:
+$ hub clone jkcfg/jk
+
+$ cd ./jk
+```
+
+Then pull most of the dependencies using the `Makefile`:
+```bash
+$ make dep
+```
+
+`jk` is linked against [`v8worker2`](https://github.com/jkcfg/v8worker2). As building V8 takes ~30 minutes, [prebuilt versions](https://github.com/jkcfg/prebuilt) are provided for `linux/amd64` and `darwin/amd64`. This also includes `flatc` from `flatbuffers`:
+```bash
+# download the prebuilt artifacts from GitHub:
+$ git clone https://github.com/jkcfg/prebuilt.git
+$ cd ./prebuilt
+
+# x64 Linux:
+$ make install-linux-amd64
+
+# x64 macOS:
+$ make install-darwin-amd64
+```
+
+### Building
+After setting up the environment, the `jk` binary can be built:
+```bash
+$ make jk
+$ ./jk --help # confirm it works
+```
+
+### License
+This project uses the [Apache License 2.0](LICENSE)
+```
+Copyright 2018,2019 Damien Lespiau <damien.lespiau@gmail.com>
+Copyright 2018,2019 Michael Bridgen <mikeb@squaremobius.net>
+Copyright 2019 Weaveworks Ltd.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+```
```

Comment:
Does this need to be reiterated here? There's the `LICENSE` file already?

Reply:
Seconded, we don't need to re-include the license here.

Reply:
Removed